spec/acceptance/nodesets/rhel-8.yml

```diff
@@ -0,0 +1,20 @@
+HOSTS:
+  rhel8:
+    roles:
+      - agent
+    platform: el-8-x86_64
+    hypervisor: docker
+    image: roboxes/rhel8
```

Comment:
could you add a comment here that points to the BKR ticket that explains we are not using an official image because we cant until that ticket is resolved.